setup.py

```diff
@@ -42,6 +42,7 @@
               'snapcraft.internal',
               'snapcraft.internal.cache',
               'snapcraft.internal.deltas',
+              'snapcraft.internal.pluginhandler',
```

Comment:
"plugin handler" was a good class name, but are we sure we want to continue using it for the package?